src/kms/Util.ts

```diff
@@ -108,3 +109,17 @@ export const extractDocumentHeaderFromStream = (
         };
         inputStream.on("readable", onRead);
     }).flatMap((maybeHeader) => (maybeHeader ? verifyAndCreateDocHeaderPb(maybeHeader) : Future.of(undefined)));
+
+/**
+ * Ensure that the deriveKeyResponse indicates that the tenant has a primary KMS config, otherwise error.
+ */
+export const verifyHasPrimaryConfig = (
+    deriveKeyResponse: DeriveKeyResponse,
```

Comment:
You could make this generic for things that have `hasPrimaryConfig` and write a test for it as well.

Reply:
Done